src/style.css

```diff
@@ -0,0 +1,38 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
```

Comment:
```suggestion
  background-position: center;
```

Reply:
thanks, I got it, henceforth I will utilize this technique)